services/galley/src/Galley/API/MLS/Message.hs

```diff
@@ -475,7 +476,15 @@ postMLSMessageToLocalConv ::
   Sem r [LocalConversationUpdate]
 postMLSMessageToLocalConv qusr senderClient con smsg lcnv = case rmValue smsg of
   SomeMessage tag msg -> do
-    conv <- getLocalConvForUser qusr lcnv
+    gtc <- getGlobalTeamConversationById lcnv
+    conv <- case gtc of
+      Just conv -> do
+        when (isNothing (gtcCreator conv)) $ do
+          setGlobalTeamConversationCreator conv (qUnqualified qusr)
+        localMembers <- getLocalMembers (qUnqualified . gtcId $ conv)
+        pure $ gtcToConv conv localMembers
+      Nothing ->
+        getLocalConvForUser qusr lcnv
```

Comment:
Wouldn't it be better to get the conversation first, then check if it's the GTC looking at its type? That way the normal case of conversations doesn't have to go through yet another db hit.

Reply:
Actually this code is now outdated, since I've changed how getting members work further down the chain, we can basically revert this.